code/modules/mob/living/carbon/xenomorph/castes/behemoth/abilities_behemoth.dm

```diff
@@ -0,0 +1,1463 @@
+	new /obj/effect/temp_visual/behemoth/landslide/hit(get_turf(src))
+
+// Attacking an Earth Pillar as a xeno has a few possible interactions, based on intent:
+// - Harm intent will reduce a counter in this structure. When the counter hits zero, the structure is destroyed, meaning it is much easier to break it as a xeno.
+// - Help intent as a Behemoth will trigger an easter egg. Does nothing, just fluff.
+/obj/structure/earth_pillar/attack_alien(mob/living/carbon/xenomorph/xeno_user, isrightclick = FALSE)
+	var/current_turf = get_turf(src)
+	switch(xeno_user.a_intent)
+		if(INTENT_HARM)
+			if(attacks_to_destroy <= 1)
+				xeno_user.do_attack_animation(src)
+				xeno_user.balloon_alert(xeno_user, "Destroyed")
+				new /obj/effect/temp_visual/behemoth/landslide/hit(current_turf)
+				new /obj/effect/temp_visual/behemoth/earth_pillar/broken(current_turf)
+				Destroy()
+				return TRUE
+			attacks_to_destroy--
+			xeno_user.do_attack_animation(src)
+			do_jitter_animation(jitter_loops = 1)
+			playsound(src, pick(possible_hit_sounds), 40)
+			xeno_user.balloon_alert(xeno_user, "Attack [attacks_to_destroy] more time(s) to destroy")
+			new /obj/effect/temp_visual/behemoth/landslide/hit(current_turf)
+			return TRUE
+		if(INTENT_HELP)
+			if(isxenobehemoth(xeno_user))
+				xeno_user.do_attack_animation(src)
+				do_jitter_animation(jitter_loops = 1)
+				playsound(src, 'sound/effects/behemoth/earth_pillar_eating.ogg', 10, TRUE)
+				xeno_user.visible_message(span_xenowarning("\The [xeno_user] eats away at the [src.name]!"), \
+				span_xenonotice(pick(possible_eating_messages)), null, 5)
+				return TRUE
+			return FALSE
+
+/obj/structure/earth_pillar/ex_act(severity)
+	switch(severity)
+		if(EXPLODE_DEVASTATE)
+			Destroy()
+		if(EXPLODE_HEAVY)
+			take_damage(max_integrity / 2)
+		if(EXPLODE_LIGHT)
+			take_damage(max_integrity / 3)
+
+/// Seismic Fracture (as in the ability) has a special interaction with any Earth Pillars caught in its attack range.
+/// Those Earth Pillars will reflect the same attack in a similar range around it, destroying itself afterwards.
+/obj/structure/earth_pillar/proc/seismic_fracture()
+	if(warning_flashes <= 0)
+		new /obj/effect/temp_visual/behemoth/earth_pillar/destroyed(get_turf(src))
+		qdel(src)
+		return
+	warning_flashes--
+	addtimer(CALLBACK(src, PROC_REF(seismic_fracture)), 1 SECONDS)
+	animate(src, color = "#FF6A00", time = 0.5 SECONDS, easing = CIRCULAR_EASING|EASE_OUT, flags = ANIMATION_PARALLEL)
+	animate(color = "#FFFFFF", time = 0.5 SECONDS, easing = CIRCULAR_EASING|EASE_IN, flags = ANIMATION_PARALLEL)
+
+// Earth Riser is capable of interacting with existing Earth Pillars to fire a projectile.
+// See the Earth Riser ability for specifics.
+/datum/ammo/xeno/earth_pillar
+	name = "rock"
+	icon_state = "earth_pillar"
+	sound_hit = 'sound/effects/behemoth/earth_pillar_destroyed.ogg'
+	sound_bounce = 'sound/effects/behemoth/earth_pillar_destroyed.ogg'
+	ping = null
```

Comment:
Dunno which one of these three lines you mean, but I'll provide an explanation for all three:
`sound_hit`: This is the sound played when the projectile hits someone. I did this intentionally.
`sound_bounce`: Unneeded since this is not a bouncing projectile. Removed.
`ping`: This is an animation that plays when the projectile hits something. It is nulled because it's otherwise an acid-looking icon, which is unfitting for this.

Reply:
Talked with Tivi in Discord about this, there was a misunderstanding regarding what `ping` did. Resolved.